src/sections/channels.tsx

```diff
@@ -0,0 +1,492 @@
+            closing: false,
+            closeSuccess: false,
+            closeErrors: [
+              ...(prevStates[channelId]?.closeErrors || []),
+              {
+                error: e.error,
+                status: e.status,
+              },
+            ],
+          },
+        }));
+      });
+  };
+
+  const fundChannel = (channelId: string, peerId: string) => {
+    return (
+      <>
+        <button onClick={() => handleOpenFundingPopup(channelId)}>Fund</button>
+        <Dialog
+          open={openFundingPopups[channelId] || false}
+          onClose={() => closeFundingPopup(channelId)}
+        >
+          <DialogTitle>Fund Channel</DialogTitle>
+          <DialogContent>
+            <TextField
+              label="Funding Amount"
+              type="number"
+              value={fundingAmount}
+              onChange={(e) => set_fundingAmount(e.target.value)}
+            />
+          </DialogContent>
+          <DialogActions>
+            <button onClick={() => closeFundingPopup(channelId)}>Cancel</button>
+            <button
+              onClick={() => handleFundChannels(peerId, channelId)}
+              disabled={!fundingAmount || parseFloat(fundingAmount) <= 0}
+            >
+              Fund
+            </button>
+          </DialogActions>
+        </Dialog>
+      </>
+    );
+  };
+
+  const handleFundChannels = (peerId: string, channelId: string) => {
+    const parsedOutgoing =
+      parseFloat(fundingAmount ?? '0') >= 0 ? fundingAmount ?? '0' : '0';
+
+    dispatch(
+      actionsAsync.fundChannelsThunk({
+        apiEndpoint: loginData.apiEndpoint!,
+        apiToken: loginData.apiToken!,
+        peerId: peerId,
+        incomingAmount: '0',
+        outgoingAmount: parsedOutgoing,
+      })
+    )
+      .unwrap()
+      .then(() => {
+        set_fundingStates((prevStates) => ({
+          ...prevStates,
+          [channelId]: {
+            funding: false,
+            fundingSuccess: true,
+            fundingErrors: [],
+          },
+        }));
+        handleRefresh();
+      })
+      .catch((e) => {
+        set_fundingStates((prevStates) => ({
+          ...prevStates,
+          [channelId]: {
+            funding: false,
+            fundingSuccess: false,
+            fundingErrors: [
+              ...(prevStates[channelId]?.fundingErrors || []),
+              {
+                error: e.error,
+                status: e.status,
+              },
+            ],
+          },
+        }));
+      })
+      .finally(() => {
+        closeFundingPopup(channelId);
+      });
+  };
+
+  return (
+    <Section className="Channels--aliases" id="Channels--aliases" yellow>
+      <h2>
+        Channels
+        <button onClick={handleRefresh}>Refresh</button>
+      </h2>
+      <Box sx={{ borderColor: 'divider' }}>
+        <Tabs value={tabIndex} onChange={handleTabChange} centered>
+          <Tab label="Incoming" />
+          <Tab label="Outgoing" />
+        </Tabs>
+      </Box>
```

Comment:
You need to handle the params in the url here. If we go on the Channels subpage, you lose the login credentials from the omnibar